umh.docs.umh.app/content/en/docs/features/data-connectivity-node-red/index.md

```diff
@@ -3,31 +3,78 @@ title = "Data connectivity with Node-RED"
 menuTitle = "Data connectivity with Node-RED"
 description = ""
 weight = 4
-draft = true
+draft = false
 +++
 
-Introduction sentence
+One feature of the UMH is to connect devices on the shopfloor such as PLCs, Quality Stations or MES / ERP systems with the Unified Namespace using Node-RED. 
+Node-RED has a large library of nodes, which lets you connect various protocols. It also has a user-friendly UI
+with little code, making it easy to configure the desired nodes.
+
+![](/images/features/ifm-retrofitting/noderedDifferentProtocols.png?width=60%)
 
 ## When should I use it?
 
+Sometimes it is necessary to connect a lot of different protocols (e.g Siemens-S7, OPC-UA, Serial, ...) and node-RED can be a maintainable
+solution to connect all these protocols without the need for other data connectivity tools. Node-RED is largely known in 
+the IT/OT-Community making it a familiar tool for a lot of users.
 
 ## What can I do with it?
 
 By default, you can
 - connect to MQTT using the MQTT node
 - connect to HTTP using the HTTP node
+- connect to TCP using the TCP node
+- connect to IP using the UDP node
 
 By installing additional packages, you can:
-- connect to OPC-UA (node-red-controb-opcua)
-- ...
+- connect to OPC-UA (node-red-contrib-opcua)
+- connect to kafka (node-red-contrib-kafkajs)
+- connect to Siemens-S7 (node-red-contrib-s7)
+- connect to serial (node-red-node-serialport)
+- connect to modbus (node-red-contrib-modbus had)
+- connect to MC-protocol (node-red-contrib-mcprotocol)
+- connect to OMRON FINS Ethernet protocol (node-red-contrib-omron-fins)
+- connect to EtherNet/IP Protocol (node-red-contrib-cip-ethernet-ip)
+- connect to PostgreSQL (node-red-contrib-postgresql)
+- connect to SAP SQL Anywhere
```

Comment:
Really ugly to read. Try something on the line of this:

By default there connector nodes for common protocols:
- MQTT
- HTTP
- TCP
- IP

Furthermore, you can install packages to support more connection protocols. For example:
- OPC-UA (node-red-contrib-opcua)
- ...